### ISSUE TITLE
Nobrute Report - Fails to run

### ISSUE THREAD

**Issue by mero01** [1N3/Sn1per#33]
Hey Mate, 

Using the **"nobrute report"** option the script fails. You are presented with the following errors. Obviously the nmap does not run, thus finding no ports and the rest of the script exits without completion. 

`Running the "nobrute" option **without** the "report" - Everything works as per expected. 

```
`################################### Running port scan ##############################

Starting Nmap 7.12 ( https://nmap.org ) at 2016-04-24 05:26 EDT
Found no matches for the service mask 'report' and your specified protocols
QUITTING!

Starting Nmap 7.12 ( https://nmap.org ) at 2016-04-24 05:26 EDT
Found no matches for the service mask 'report' and your specified protocols
QUITTING!

################################### Running Intrusive Scans ########################
################################## Skipping Brute Force ############################
################################### Done!`
```
Looking into the code, it appears you may be missing the following entry for the "nobrute" option/mode. 
```
**if [ "$OPT1" = "report" ]; then**
./sniper $TARGET $MODE | tee ./loot/sniper-$MODE-`date +%Y%m%d%H%M`.txt 2>&1
```

**Comment by 1N3** [comment]
Thanks, will fix this.

**Comment by 1N3** [comment]
Fixed in 1.6e! Cheers!
